sentry-opentelemetry/sentry-opentelemetry-core/build.gradle.kts

```diff
@@ -0,0 +1,74 @@
+import net.ltgt.gradle.errorprone.errorprone
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    `java-library`
+    kotlin("jvm")
+    jacoco
+    id(Config.QualityPlugins.errorProne)
+    id(Config.QualityPlugins.gradleVersions)
+}
+
+configure<JavaPluginExtension> {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+tasks.withType<KotlinCompile>().configureEach {
+    kotlinOptions.jvmTarget = JavaVersion.VERSION_1_8.toString()
+}
+
+dependencies {
+    compileOnly(projects.sentry)
+
+    compileOnly("io.opentelemetry:opentelemetry-sdk:${Config.Libs.otelVersion}")
```

Comment:
I guess we'd need some docs or something around that, so people don't expect to add sentry and that it supports OTel out of the box

Reply:
Yes, there'll need to be some OTEL specific docs that suggest how you can combine Sentry and OTEL, e.g. by using our agent JAR. I already have it on the TODOs here https://github.com/getsentry/sentry-java/issues/2327